### PR TITLE
ShA/ add unique name validation for voteItem

### PR DESCRIPTION
**Pull request by AlekseiiShevchuk**

### CODE REVIEW

File: app/Http/Requests/VoteItemRequest.php [hunk 1]
```diff
@@ -21,9 +23,20 @@ public function authorize()
...
+                foreach ($existedVoteItems as $voteItem){
+                    $stringOfVoteItemsNames .= $voteItem['name'] . ',';
+                }
+            }
+
+        };
         return [
             'vote_id' => 'required|exists:votes,id|integer',
-            'name' => 'required',
+            'name' => 'required|not_in:' . $stringOfVoteItemsNames,
```
Comment (binaryvo) [inline app/Http/Requests/VoteItemRequest.php]: I guess that will be error with $stringOfVoteItemsNames if $this->vote_id equals 0 or vote with id is not presented in db

Comment (AlekseiiShevchuk) [reply app/Http/Requests/VoteItemRequest.php]: i fixed it

File: app/Http/Requests/VoteItemRequest.php [hunk 2]
```diff
@@ -21,9 +23,20 @@ public function authorize()
      */
     public function rules()
     {
+        $stringOfVoteItemsNames ='';
+        if($this->vote_id){
+            $vote = Vote::find($this->vote_id);
+            if($vote){
+                $existedVoteItems = $vote->voteitems()->get()->toArray();
+                foreach ($existedVoteItems as $voteItem){
+                    $stringOfVoteItemsNames .= $voteItem['name'] . ',';
```
Comment (TatsuUkraine) [inline app/Http/Requests/VoteItemRequest.php]: this row will eventually create row like "1,2,3," ,change it to avoid "," at the end

**Comment by AlekseiiShevchuk** [comment]
Done

**Pull request merged**